crates/resource-resolver/README.md

```diff
@@ -0,0 +1,8 @@
+# cargo-packager-resource-resolver
+
+Most of the code in this crate comes from [tauri-utils](https://github.com/tauri-apps/tauri/tree/dev/core/tauri-utils).
```

Comment:
We should remove this and document what this crate does and how to use it. We should also mention it in `cargo-packager` rust docs and README.md